content/en/docs/Creating derivatives/build_iso.md

```diff
@@ -76,182 +47,128 @@ label: "COS_LIVE"
-
-### `rootfs_image`
-
-A container image reference (e.g. `quay.io/.../...:latest`) that will be used as a rootfs for the ISO.
-
-### `packages.rootfs`
+Packages or sources can be a Luet package (as in the example), an image reference (then an explicit tag is required) or a local path. Sources are stacked in the given order, so one can easily overwrite or append data by simply adding a local path as the last source.
 
-A list of luet packages to install in the rootfs. The rootfs will be squashed to a `rootfs.squashfs` file
+### Command flags
 
-### `packages.uefi`
+- **name**: Name of the ISO image. It will be used to generate the `*.iso` file name
+- **output**: Path of the destination folder of created images
+- **date**: If present it includes the date in the generated file name
+- **overlay-rootfs**: Sets the path of a tree to overlay on top of the system root-tree
+- **overlay-uefi**: Sets the path of a tree to overaly on top of the EFI image root-tree
+- **overlay-iso**: Sets the path of a tree to overlay on top of the ISO filesystem root-tree
+- **label**: Sets the volume label of the ISO filesystem
+- **repo**: Sets the URI of a repository to include together with the repositores set in manifest or the default one if no repositories are set in manifest. This option can be set multiple times.
 
-A list of luet packages to be present in the efi ISO sector.
-
-### `packages.isoimage`
-
-A list of luet packages to be present in the ISO image.
-
-### `repository.packages`
-
-A list of package repository (e.g. `repository/mocaccino-extra`) to be installed before `luet install` commands Requirements
+## Configuration reference
 
-### `initramfs.kernel_file`
+### `iso.rootfs`
 
-The kernel file under `/boot/` that is your running  kernel. e.g. `vmlinuz` or `bzImage`
+A list of sources (luet package, container image or local path) to install in the rootfs. The rootfs will be squashed to a `rootfs.squashfs` file
 
-### `initramfs.rootfs_file`
+### `iso.uefi`
 
-The initrd file under `/boot/` that has all the utils for the initramfs
+A list of sources (luet package, container image or local path) to install in the efi FAT image or partition.
 
-### `image_prefix`
+### `iso.image`
 
-ISO image prefix to use
+A list of sources (luet package, container image or local path) to install in ISO filesystem.
 
-### `image_date`
+### `iso.label`
 
-Boolean indicating if the output image name has to contain the date
+The label of the ISO filesystem. Defaults to `COS_LIVE`. Note this value is tied with the bootloader and kernel paramters to identify the root device.
 
-### `image_name`
+### `repositories`
 
-A string representing the ISO final image name
+A list of Luet package repositories
 
-### `arch`
+### `repositories.uri`
 
-A string representing the arch. Defaults to `x86_64`.
+The URI of the repository, it is the only mandatory value for a repository. Repository type (`docker`, `disk` or `http`) is guessed from this URI if not provided.
```

Comment:
There is text for the repo type here which should be below?

Reply:
ah no, this is a clarification that if type is not provided, sorry, read it wrong.